curvefs/src/client/s3/client_s3_adaptor.cpp

```diff
@@ -123,7 +134,10 @@ int S3ClientAdaptorImpl::Write(uint64_t inodeId, uint64_t offset,
         }
     }
     int ret = fileCacheManager->Write(offset, length, buf);
-    VLOG(6) << "write end inodeId:" << inodeId << ",ret:" << ret;
+    pendingReq_.fetch_sub(1, std::memory_order_seq_cst);
+    fsCacheManager_->DataCacheByteDec(length);
```

Comment:
Why we DataCacheByteDec here when fileCacheManager write succeed.

Reply:
just release DataCacheByteInc before.